monai/transforms/utils.py

```diff
@@ -1172,6 +1178,168 @@ def get_largest_connected_component_mask(
     return convert_to_dst_type(out, dst=img, dtype=out.dtype)[0]
 
 
+def get_largest_connected_component_mask_point(
+    img_pos: Tensor,
+    img_neg: Tensor,
+    point_coords: Tensor,
+    point_labels: Tensor,
+    pos_val: Sequence[int] = (1, 3),
+    neg_val: Sequence[int] = (0, 2),
+    margins: int = 3,
+) -> Tensor:
+    """
+    Gets the largest connected component mask of an image that include the point_coords.
+    # TODO: need author to provide more details about this function. Especially about each argument.
```

Comment:
Hi @heyufan1995 , can you help to finish the doc strings for this function?

Reply:
I've updated the doc in my pr. The img_pos/img_neg input should be [B, 1, H, W, D]
    """
    Gets the connected component of img_pos and img_neg that include the positive points and 
    negative points separately. The function is used for combining automatic results with interactive
    results in VISTA3D. 
    Args:
        img_pos: bool type array. [B, 1, H, W, D]. B foreground masks from a single 3D image.
        img_neg: same format as img_pos but corresponds to negative points.
        pos_val: positive point label values.  
        neg_val: negative point label values.
        point_coords: [B, N, 3]
        point_labels: [B, N]
    """